eventbus/jetstream/sensor/sensor_jetstream.go

```diff
@@ -55,12 +55,22 @@ func (stream *SensorJetstream) Initialize() error {
 	}
 	// create Key/Value store for this Sensor (seems to be okay to call this if it already exists)
 	stream.keyValueStore, err = stream.MgmtConnection.JSContext.CreateKeyValue(&nats.KeyValueConfig{Bucket: stream.sensorName})
-	if err != nil {
+	switch err {
```

Comment:
Can we query first, if it's not existing, then create it?

Reply:
done